digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/AddressLinkBuilder.scala

```diff
@@ -125,16 +125,17 @@ trait AddressLinkBuilder {
   }
 
   def fuseRoadAddress(roadAddresses: Seq[RoadAddress]): Seq[RoadAddress] = {
```

Comment:
Add comment to do it on the apply changes user story

Reply:
Comment added. (This pull request is already so big that Github doesn't show the latest changes any more)